src/test/java/com/fishercoder/_66Test.java

```diff
@@ -8,28 +9,44 @@
 
 public class _66Test {
     private static _66.Solution1 solution1;
+    private static _66.Solution2 solution2;
     private static int[] digits;
 
     @BeforeClass
     public static void setup() {
         solution1 = new _66.Solution1();
+        solution2 = new _66.Solution2();
     }
 
     @Test
     public void test1() {
         digits = new int[]{9, 9, 9, 9};
         assertArrayEquals(new int[]{1, 0, 0, 0, 0}, solution1.plusOne(digits));
+        digits = new int[]{9, 9, 9, 9};
```

Comment:
Please write new tests for `solution2` so each test is responsible for only one solution. Thanks